accelerator/http.c

```diff
@@ -34,16 +34,21 @@ static status_t ta_http_url_matcher(char const *const url, char *const regex_rul
     return SC_HTTP_NULL;
   }
   regex_t reg;
+  regmatch_t pmatch;
```

Comment:
Why did you call it `pmatch`? I would say "pattern."

Reply:
In the manual page from `man regexec` call it pmatch or pmatch[].

I think it means the position of match patterns.

`int regexec(const regex_t *preg, const char *string, size_t nmatch,
                   regmatch_t pmatch[], int eflags);
`

> Byte offsets
       Unless REG_NOSUB was set for the compilation of the pattern buffer, it is possible to obtain match addressing information.   pmatch  must  be
       dimensioned  to  have  at least nmatch elements.  These are filled in by regexec() with substring match addresses.  The offsets of the subex‐
       pression starting at the ith open parenthesis are stored in pmatch[i].  The  entire  regular  expression's  match  addresses  are  stored  in
       pmatch[0].   (Note  that  to return the offsets of N subexpression matches, nmatch must be at least N+1.)  Any unused structure elements will
       contain the value -1.

       The regmatch_t structure which is the type of pmatch is defined in <regex.h>.

           typedef struct {
               regoff_t rm_so;
               regoff_t rm_eo;
           } regmatch_t;

       Each rm_so element that is not -1 indicates the start offset of the next largest substring match within the string.  The relative rm_eo  ele‐
       ment indicates the end offset of the match, which is the offset of the first character after the matching text.